src/amo/components/RatingManager/index.js

```diff
@@ -28,10 +34,12 @@ import {
 import translate from 'core/i18n/translate';
 import log from 'core/logger';
 import Button from 'ui/components/Button';
+import { genericType, successType } from 'ui/components/Notice';
+import RatingManagerNotice from 'ui/components/RatingManagerNotice';
```

Comment:
This seems like a pretty specific component so I'd expect it in `amo/components`. I guess it doesn't really matter, though.

Reply:
Sure, I can move it.